src/main/java/seedu/agendum/logic/LogicManager.java

```diff
@@ -20,17 +23,26 @@
 
     private final Model model;
     private final Parser parser;
+    private CommandLibrary commandLibrary;
 
-    public LogicManager(Model model, Storage storage) {
+    public LogicManager(Model model, Storage storage, CommandLibrary commandLibrary) {
```

Comment:
I didn't want to use singleton/static variables when I can ensure that they are all the same commandLibrary this way. Easier for the test cases